### DISCUSSION TITLE
Can't connect to spotify

### DISCUSSION THREAD

**Discussion by XDawn66** [judahpaul16/gpt-home#63] [Q&A]
Hello,
I having trouble connecting my decvice to the spotify api. I tried to follow the guide by inputing this as the spotify Redirect URIs
```
http://gpt-home.local/api/callback

```
but I was told that 
INVALID_CLIENT: Invalid redirect URI.
Then, I tried the second Redirect URIs by inputing:
http://192.168.1.32/api/callback,
Then I got,
Internal Server Error.
I as tried using the local host 3000 or 80  + /callback, but they still gave me the INVALID_CLIENT: Invalid redirect URI. error.

I beleive my Client ID and Client secret is all correct. Does anyone know what's the issue? Thanks

**Comment by judahpaul16** [comment]
This appears to be related to #36
![image](https://github.com/user-attachments/assets/815e71c3-230d-4baf-8603-ed7762abdae5)

**Reply by XDawn66** [reply]
Thanks for letting me know! I will look into that and try to help as well.
